rpcd/etc/openstack_deploy/env.d/nova.yml

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
A substitute may also be targeting the MaaS playbook at `neutron_l3_agent:neutron_metadata_agent:!compute_hosts`